Framework/BrowserSettings/ConfigReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Framework.BrowserSettings
+{
+    class ConfigReader
```

Comment:
Can we use this class on model settings?

Reply:
Yes, class is public and method is static. Class could be static also, but we need to discuss